src/Page/SubmitStory/View.elm

```diff
@@ -0,0 +1,121 @@
+module Page.SubmitStory.View exposing (..)
```

Comment:
Do we expose everything on the other page views?

Reply:
noop, fixed